opds.py

```diff
@@ -1598,16 +1590,27 @@ def acquisition_link(cls, rel, href, types, active_loan=None):
         # and will not ask patrons to enter their passphrases
         # For more information please look here:
         # https://readium.org/lcp-specs/notes/lcp-key-retrieval.html#including-a-hashed-passphrase-in-an-opds-1-catalog
-        if active_loan and active_loan.license_pool.collection.protocol == ExternalIntegration.LCP:
+        if active_loan and active_loan.license_pool.collection.protocol in [
```

Comment:
It would be nice to more elegantly hook into this, rather then just have a big list of external integrations in an if statement. Could we put this back on the protocol perhaps or the licence_pool.

Reply:
Does the loan need to be active in order for us to add this to the feed? We probably already know what the password is before the loan is active right? This might save the mobile apps an extra request.

Reply:
Yes, unfortunately, the loans needs to be active because `hashed_passphrase` is a sensitive piece of information and should be shown only to authenticated clients. The public CM's feed is not appropriate since it doesn't require authentication but the loans does and it seems to be a better place.